send-emails-from-a-service/README.md

```diff
@@ -54,7 +54,7 @@ The flow is as follows.
 }
 ```
 
-3. Call the payment backend service to make the payment and retrieve the reservation response which will have a payload similar to that shown below. If the payment is successful, send an email to the user with the appointment details.
+3. Call the payment backend service to make the payment and retrieve the reservation response which will have a payload similar to the following. If the payment is successful, send an email to the user with the appointment details.
```

Comment:
Why do we need this change?

Reply:
Thought it would be better to use `following` since that's what we've used everywhere else

Reply:
Would have been OK if we didn't have the next sentence. But since we have it I prefer the former version.